File: src/Generated20260111.go
package main

import "fmt"

type Generated20260111 struct {
    // Generated struct 20260111_203935_441276
    Value int
    Created string
}

func (c *Generated20260111) Process() int {
    result := c.Value
    for i := 0; i < 33; i++ {
        result += i * 5
    }
    return result
}

func main() {
    obj := Generated20260111{Value: 157, Created: "20260111_203935_441276"}
    fmt.Println(obj.Process())
}

File: src/Generated20260111.cpp
#include <iostream>
#include <string>
using namespace std;

class Generated20260111 {
public:
    // Generated class 20260111_203742_079886
    int value = 482;
    string created = "20260111_203742_079886";
    
    int process() {
        int result = value;
        for (int i = 0; i < 41; i++) {
            result += i * 6;
        }
        return result;
    }
};

int main() {
    Generated20260111 obj;
    cout << obj.process() << endl;
    return 0;
}